src/Stratis.Bitcoin.IntegrationTests/Wallet/WalletAddressGenerationAndFundsVisibility_Steps.cs

```diff
@@ -34,14 +35,14 @@ public partial class Wallet_address_generation_and_funds_visibility : BddSpecifi
 
         protected override void BeforeTest()
         {
-            this.nodeGroupBuilder = new NodeGroupBuilder(Path.Combine(this.GetType().Name, this.CurrentTest.DisplayName), KnownNetworks.RegTest);
+            this.nodeGroupBuilder = new NodeGroupBuilder(Path.Combine(this.GetType().Name, this.CurrentTest.DisplayName), NetworkRegistration.Register(new TestRegTest()));
             this.sharedSteps = new SharedSteps();
         }
 
         protected override void AfterTest()
         {
-            this.sendingStratisBitcoinNode.FullNode.Network.Consensus.CoinbaseMaturity = this.previousCoinBaseMaturity;
-            this.receivingStratisBitcoinNode.FullNode.Network.Consensus.CoinbaseMaturity = this.previousCoinBaseMaturity;
+            ((MutableTestConsensus) this.sendingStratisBitcoinNode.FullNode.Network.Consensus).CoinbaseMaturity = this.previousCoinBaseMaturity;
```

Comment:
Are these the only integration tests that currently fail if you use an immutable CoinBaseMaturity? I feel like there should be others.

Reply:
These are the only ones that modify CoinBaseMaturity.